src/lib/sidebar/InterventionList.svelte

```diff
@@ -17,8 +17,11 @@
       return feature.properties.planning?.name || "Untitled polygon";
     }
     if (schema == "v2") {
+      // TODO Revisit auto-generated enum types
```

Comment:
Still needs more work to figure this out, but I  don't think it's worth blocking better checks